lighthouse-core/report/html/report-styles.css

```diff
@@ -213,7 +213,8 @@
   word-break: break-word;
   color: hsl(174, 100%, 27%);
 }
-.lh-node:hover {
+.lh-node:hover,
+div:target {
```

Comment:
seems a bit weird to share the style with HTML nodes, let's split this out into its own style with a comment about the scroll to behavior :)

Reply:
Sounds reasonable to me. I just didn't find a place where to put it at first *g*